### ISSUE TITLE
Doesnt Boot

### ISSUE THREAD

**Issue by S1m1B1lly** [PGBSean/Surface-Pro-5-OpenCore#3]
Hangs on apple logo after picking and reboots

**Comment by PGBSean** [comment]
Please check one of the following:
+ You have Secure Boot disabled in your BIOS
+ LauncherOption is disabled
+ ScanPolicy is set to 0

Newer releases aren't properly checked so there will be some toggles that may affect the booting sequence on newer machines. Sorry about that!

**Comment by Leventionz** [comment]
Similar issue here, first menu appears, I select it then apple logo appears and that's it.

Using Somoma BaseSystem.dmg with OC 0.9.6 and your latest 1.2 EFI with the 1.21 config.plist
I have used genSMBIOS to update the various serials.

Have checked the above options are as you state also.

**Comment by LordImmaculate** [comment]
Are you sure you are on an Suface Pro 5? I'm not sure if it would make a difference with the cpu (probably).

**Comment by Leventionz** [comment]
Model is 1796, when I have hunted down the specs it looks to be a 5 yes.

i7-7660U @ 2.5 with a 640 Plus Integrated Graphics.

https://support.microsoft.com/en-us/surface/surface-pro-5th-gen-specs-and-features-42d321e4-52d6-dcb1-e014-9ffc76fbca14

**Comment by PGBSean** [comment]
> ..1.2 EFI with the 1.21 config.plist

This should explain why your build isn't bootable. The 1.2 EFI use the OC0.9.5 package, while the 1.2.1 plist use OC0.9.6

**Comment by Leventionz** [comment]
Thanks, managed to format the entire drive to APFS and install using the EFI 1.2 with OC0.9.5 Debug.

Post install everything works up until entering a username or accepting the EULA then it freezes.

Is there any way I can debug or provide more information to you as to why this might be happening. Tried using Ventura but can try a different version if needed.

**Comment by Leventionz** [comment]
Thank you for these instructions. I have tried deleting 4. and also removing all entries under 3.
Unfortunately now what happens is the Apple logo is displayed, it begins booting but then freezes about 1/3 of the way through the loading progress bar.

Separately I have tried a fresh internet recovery install using Sonoma and get the same results.

**Comment by Leventionz** [comment]
Managed to get it up and running. Used 000025A8 for AAPL,ig-platform-id and 25A80000 for device-id

It's running slowly and text is the too small.

Followed guide here for Iris Plus https://github.com/acidanthera/WhateverGreen/blob/master/Manual/FAQ.IntelHD.en.md

I'm sure it can be improved but I have never done this before so lots of learning to do.

**Comment by PGBSean** [comment]
At this moment, you have to **refer** to the GPU Patching guide on the OpenCore Post-Install guide. You may also consider **changing** your SMBIOS to the one that matches your current GPU for better results.

**Comment by Leventionz** [comment]
Fixed!

AAPL,ig-platform-id = 02002659
device-id = 26590002

**Comment by LordImmaculate** [comment]
@Leventionz Can you close the issue when it's fixed please? BTW good job fixing it!

**Comment by Leventionz** [comment]
Will do apologies. Separately I have noted for the Iris Plus 640 that support stops at Ventura. Whilst this can be worked around by using the SystemProductName 'MacBookPro15,2' as you have done, you lose Metal support and overall performance is impacted.

I have modified config.plist to use MacBookPro14,1 which further improves performance.

The limitation is you cannot use Sonoma. To workaround this I plan to use OCLP; if successful I can merge my updates into this repository if you would like for others to use.

**Comment by PGBSean** [comment]
This should be resolved by the latest release in https://github.com/PGBSean/Surface-Pro-5-OpenCore/releases/tag/20231202

**Comment by martijn-brant** [comment]
> Leventionz

[Leventionz](https://github.com/Leventionz) how did you get to those numbers? I've tried these and aren't working well for me. I've also tried a bunch of other calculated values using the linked guide in this thread but can't seem to get it working. It's either correct-DPI but slow/unstable or 100% scaled high res (as in, everything is ultra tiny). I'm using the i7 model of the SP5.

Tried MacbookPro14 and 15 SMBIOS references as well.
